apps/site/src/views/Home/sections/FAQ/FAQ.module.scss

```diff
@@ -0,0 +1,121 @@
+@use "zothacks-theme" as theme;
+@use "bootstrap-utils" as utils;
+
+.faq {
+	position: relative;
+	display: flex;
+	flex-direction: column;
+
+	margin-left: 40px;
+	margin-right: 40px;
+	padding-bottom: 30px;
+
+	background-color: theme.$beige;
+}
+
+.title {
+	align-self: center;
+	margin-top: 24px;
+	margin-bottom: 24px;
+}
+
+.vertical-line {
+	position: absolute;
+	justify-self: flex-start;
+
+	height: 100%;
+	width: 116px;
+	border-right: 4px solid theme.$red;
+	opacity: 0.45;
```

Comment:
Nitpick: Since just the border needs to be partly transparent, we can specify the color as such with the `rgba` mixin. This may or may not help with an underlying rendering optimization, but I have no idea given the background is already transparent.